modules/navier_stokes/test/tests/finite_volume/wcns/enthalpy_equation/enthalpy_equation-physics.i

```diff
@@ -0,0 +1,186 @@
+H = 0.015
+L = 1
+bulk_u = 0.01
+p_ref = 101325.0
+
+advected_interp_method = 'upwind'
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = ${L}
+    ymin = -${H}
+    ymax = ${H}
+    nx = 30
+    ny = 15
+  []
+[]
+
+[Problem]
+  linear_sys_names = 'u_system v_system pressure_system energy_system'
+[]
+
+[Physics]
+  [NavierStokes]
+    [FlowSegregated]
+      [flow]
+        compressibility = 'weakly-compressible'
+
+        velocity_variable = 'vel_x vel_y'
+
+        density = 'rho'
+        dynamic_viscosity = 'mu'
+
+        initial_velocity = '${bulk_u} 0 0'
+        initial_pressure = '${p_ref}'
+
+        inlet_boundaries = 'left'
+        momentum_inlet_types = 'fixed-velocity'
+        momentum_inlet_functors = '${bulk_u} 0'
+
+        wall_boundaries = 'top bottom'
+        momentum_wall_types = 'noslip noslip'
+
+        outlet_boundaries = 'right'
+        momentum_outlet_types = 'fixed-pressure'
+        pressure_functors = '${p_ref}'
+
+        orthogonality_correction = false
+        momentum_advection_interpolation = ${advected_interp_method}
+        pressure_two_term_bc_expansion = false
+        momentum_two_term_bc_expansion = false
+      []
+    []
+    [FluidHeatTransferSegregated]
+      [energy]
+        coupled_flow_physics = flow
+
+        solve_for_enthalpy = true
+        fluid_temperature_variable = 'T'
+
+        fp = 'lead'
+        thermal_conductivity = 'k'
+        specific_heat = 'cp'
+
+        initial_temperature = '777'
+        initial_enthalpy = '44000'
+
+        energy_inlet_types = 'fixed-temperature'
+        energy_inlet_functors = '860'
+        energy_wall_types = 'fixed-temperature fixed-temperature'
+        energy_wall_functors = '950 950'
+
+        energy_advection_interpolation = ${advected_interp_method}
+        energy_two_term_bc_expansion = false
+        use_nonorthogonal_correction = false
+      []
+    []
+  []
+[]
+
+[AuxVariables]
+  [T]
+    type = MooseLinearVariableFVReal
+  []
+[]
```

Comment:
The physics can auto-create this right? This is just for testing?

Reply:
not right now. Maybe later

Reply:
I'd also vote for including this in the physics.

Reply:
ok will do!

Reply:
done